mod/uexport.php

```diff
@@ -2,20 +2,26 @@
 /**
  * @file mod/uexport.php
  */
+
 use Friendica\App;
 use Friendica\Core\Hook;
 use Friendica\Core\L10n;
 use Friendica\Core\Renderer;
 use Friendica\Core\System;
 use Friendica\Database\DBA;
+use Friendica\Database\DBStructure;
 
 function uexport_init(App $a) {
+	global $dbStructure;
```

Comment:
Please add a TODO comment about moving this global variable to a static Module variable when it is moved to `src/`.